### ISSUE TITLE
Profile API Email Subscription Types Should Reflect Required List Relationship

### ISSUE THREAD

**Issue by TheDunco** [klaviyo/klaviyo-api-node#36]
Version: `6.0.0`

I noticed when trying to subscribe a profile to an email list, the TypeScript types don't currently reflect the fact that having a list relationship is REQUIRED. I only found this out by sending the request and getting back an API error that it was required. Obviously you need to specify which list to subscribe the profiles to, but the types need to reflect that too for this to be a truly developer-friendly TypeScript SDK.

The main benefit of a TypeScript SDK is to know that the shape of the data is absolutely correct before sending the request. I feel like that's a bit lost in the current TypeScript implementation overall.

Here's an example of what I'm specifically talking about:

``` Typescript
import { ListEnum, ProfileEnum, ProfileSubscriptionBulkCreateJobEnum, SubscriptionChannels } from 'klaviyo-api';
...
await klaviyo.profiles.subscribeProfiles({
                data: {
                    type: ProfileSubscriptionBulkCreateJobEnum.ProfileSubscriptionBulkCreateJob,
					// TODO: This should be a required field!!! (If an email subscription is provided)
                    relationships: {
                        list: {
                            data: {
                                id: KLAVIYO_LIST_ID,
                                type: ListEnum.List,
                            },
                        },
                    },
                    attributes: {
                        profiles: {
                            data: [
                                {
                                    type: ProfileEnum.Profile,
                                    attributes: {
                                        email,
                                        subscriptions: {
                                            email: [SubscriptionChannels.EmailEnum.Marketing],
                                        },
                                    },
                                },
                            ],
                        },
                    },
                },
            });
```

**Comment by Ian-Montgomery-Klaviyo** [comment]
Thanks for the feedback. There was an error upstream that wasn't marking this as required. This is fixed in a 6.0.1 patch you can view this change and other fixed models that shared the same issue [here](https://github.com/klaviyo/klaviyo-api-node/releases/tag/v6.0.1)

**Comment by TheDunco** [comment]
> Thanks for the feedback. There was an error upstream that wasn't marking this as required. This is fixed in a 6.0.1 patch you can view this change and other fixed models that shared the same issue [here](https://github.com/klaviyo/klaviyo-api-node/releases/tag/v6.0.1)

Sweet, thanks! Glad to hear this is already being fixed/worked on!

To be clear I think it's fantastic that Klaviyo is providing native TS support and am very happy for this feature, just want it to be the best it can be!
